internal/manifest/validate_test.go

```diff
@@ -42,9 +48,21 @@ var (
 	}
 )
 
+// TestDomain tests the domain which will be formed from node name
 func TestDomain(t *testing.T) {
 	err := checkLengthOfFutureDomain(testDomainSuccess)
 	require.NoError(t, err)
 	err = checkLengthOfFutureDomain(testDomainFail)
 	require.Error(t, err)
+	fmt.Println(err)
```

Comment:
what is the purpose of this print statement? do we further validate that the error is what we expect?

also, could this call be replaced by `testing`'s own `Log` or `Logf` so that we don't import the whole of `fmt` for this one line?